shared/react-native/ios/Keybase.xcodeproj/project.pbxproj

```diff
@@ -421,6 +421,16 @@
 			name = "Supporting Files";
 			sourceTree = "<group>";
 		};
+		02164F2C1F71EA5300F84EFC /* Recovered References */ = {
```

Comment:
I'm assuming these changes were unintentional -- can you undo before committing?